### ISSUE TITLE
Docker image vulnerabilities in v2.1.4

### ISSUE THREAD

**Issue by jmcruz1983** [sigp/lighthouse#3093]
Base image brings some vulnerabilities that need to be patched.

```
docker scan --severity high sigp/lighthouse:v2.1.4                       

Testing sigp/lighthouse:v2.1.4...

✗ High severity vulnerability found in openssl/libssl1.1
  Description: CVE-2022-0778
  Info: https://snyk.io/vuln/SNYK-UBUNTU2004-OPENSSL-2426343
  Introduced through: openssl/libssl-dev@1.1.1f-1ubuntu2.11, ca-certificates@20210119~20.04.2
  From: openssl/libssl-dev@1.1.1f-1ubuntu2.11 > openssl/libssl1.1@1.1.1f-1ubuntu2.11
  From: ca-certificates@20210119~20.04.2 > openssl@1.1.1f-1ubuntu2.11 > openssl/libssl1.1@1.1.1f-1ubuntu2.11
  From: ca-certificates@20210119~20.04.2 > openssl@1.1.1f-1ubuntu2.11
  and 1 more...
  Image layer: 'apt-get install -y --no-install-recommends libssl-dev ca-certificates'
  Fixed in: 1.1.1f-1ubuntu2.12


Organization:      
Package manager:   deb
Project name:      docker-image|sigp/lighthouse
Docker image:      sigp/lighthouse:v2.1.4
Platform:          linux/amd64
Base image:        ubuntu:20.04
Licenses:          enabled

Tested 97 dependencies for known issues, found 1 issue.

Base Image    Vulnerabilities  Severity
ubuntu:20.04  15               0 critical, 0 high, 1 medium, 14 low

Recommendations for base image upgrade:

Major upgrades
Base Image     Vulnerabilities  Severity
ubuntu:impish  13               0 critical, 0 high, 1 medium, 12 low
```

**Comment by michaelsproul** [comment]
Thanks for the report, it seems we got really unlucky with our release timing!

We've got a PR here to patch the vuln in Lighthouse (https://github.com/sigp/lighthouse/pull/3095), and once we do a release with that fix the Docker image will be automatically fixed as well (because it runs `apt-get update`).

**Comment by michaelsproul** [comment]
Confirmed that this is fixed in v2.1.5:

```
docker scan --severity high sigp/lighthouse:v2.1.5

Testing sigp/lighthouse:v2.1.5...

Organization:      michaelsproul
Package manager:   deb
Project name:      docker-image|sigp/lighthouse
Docker image:      sigp/lighthouse:v2.1.5
Platform:          linux/amd64
Base image:        ubuntu:20.04
Licenses:          enabled

✔ Tested 97 dependencies for known issues, no vulnerable paths found.

According to our scan, you are currently using the most secure version of the selected base image
```

**Comment by edaj965** [comment]
> Base image brings some vulnerabilities that need to be patched.
> 
> ```
> docker scan --severity high sigp/lighthouse:v2.1.4                       
> 
> Testing sigp/lighthouse:v2.1.4...
> 
> ✗ High severity vulnerability found in openssl/libssl1.1
>   Description: CVE-2022-0778
>   Info: https://snyk.io/vuln/SNYK-UBUNTU2004-OPENSSL-2426343
>   Introduced through: openssl/libssl-dev@1.1.1f-1ubuntu2.11, ca-certificates@20210119~20.04.2
>   From: openssl/libssl-dev@1.1.1f-1ubuntu2.11 > openssl/libssl1.1@1.1.1f-1ubuntu2.11
>   From: ca-certificates@20210119~20.04.2 > openssl@1.1.1f-1ubuntu2.11 > openssl/libssl1.1@1.1.1f-1ubuntu2.11
>   From: ca-certificates@20210119~20.04.2 > openssl@1.1.1f-1ubuntu2.11
>   and 1 more...
>   Image layer: 'apt-get install -y --no-install-recommends libssl-dev ca-certificates'
>   Fixed in: 1.1.1f-1ubuntu2.12
> 
> 
> Organization:      
> Package manager:   deb
> Project name:      docker-image|sigp/lighthouse
> Docker image:      sigp/lighthouse:v2.1.4
> Platform:          linux/amd64
> Base image:        ubuntu:20.04
> Licenses:          enabled
> 
> Tested 97 dependencies for known issues, found 1 issue.
> 
> Base Image    Vulnerabilities  Severity
> ubuntu:20.04  15               0 critical, 0 high, 1 medium, 14 low
> 
> Recommendations for base image upgrade:
> 
> Major upgrades
> Base Image     Vulnerabilities  Severity
> ubuntu:impish  13               0 critical, 0 high, 1 medium, 12 low
> ```
